qgreenland/tasks/layers/__init__.py

```diff
@@ -6,6 +6,7 @@
 from qgreenland.tasks.layers.gzipped_vector_parts import GzippedVectorParts
 from qgreenland.tasks.layers.ice_thickness_change import IceThicknessChange
 from qgreenland.tasks.layers.local_vector import LocalVector
+from qgreenland.tasks.layers.local_zipped_vector import LocalZippedVector
```

Comment:
If our code were better, we wouldn't have had to make a new pipeline for this. Continuing a pattern we already have, we could expose fetch tasks the same way we expose `INGEST_TASKS` here, and then we could mix-and-match fetch and ingest. And eventually, ingest can just be command pipelines! Maybe with exceptions. But maybe the baby step towards that baby step would be to expose fetch tasks at the configuration level. That would maybe eventually lead to us being able to get rid of the `file_type` config key? Or expose it as an `ingest_kwarg` so it's more clear what the heck it means.

Reply:
Creating a task for this.